custom/git-app-deployer/main.go

```diff
@@ -0,0 +1,183 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"github.com/litmuschaos/test-tools/pkg/log"
+	"github.com/openebs/maya/pkg/util/retry"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+)
+
+func main() {
+
+	var filePath string
+	var timeout int
+	namespace := "sock-shop"
+
+	// filePath is path for manifest, namespace is of application, timeout for application
+	filePath, timeout = GetData()
```

Comment:
filePath, timeout := GetData() and remove the pre declaration of filePath and timeout